Fix: pom amendments

matthewandrewpalmer opened the pull request:
Refactoring of POM:

- Moved versions number to properties section
- Moved  plugin configuration to plugin management

matthewandrewpalmer committed:
temporary changes

matthewandrewpalmer committed:
pom refactoring

ajorpheus commented on pom.xml:
```diff
@@ -84,39 +87,43 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <version>2.14.1</version>
```
Would it be possible to move this over to pluginManagement as well please?

ajorpheus reviewed (changes requested):
Looks good, just one minor recommendation

matthewandrewpalmer committed:
removed missed plugin

ajorpheus commented on pom.xml:
```diff
@@ -84,38 +87,46 @@
+        <plugin>
+          <groupId>com.github.ekryd.sortpom</groupId>
+          <artifactId>sortpom-maven-plugin</artifactId>
+          <version>2.8.0</version>
```
Move this version to properties section please?

ajorpheus commented on pom.xml:
```diff
@@ -84,38 +87,46 @@
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-surefire-plugin</artifactId>
+          <version>2.14.1</version>
```
Move this version to properties section please?

matthewandrewpalmer committed:
moving plugin versions to properties

Commit:
Amended configuration of sortpom plugin

Now the dependencies will be sorted by scope, artifactId and the plugins
will be sorted by artifactId. This makes is easier to locate them in the
pom.

ajorpheus reviewed (dismissed):
I have added some changes around the sortpom plugin configuration.

Looks good to me.

ajorpheus reviewed (dismissed):
Awaiting comment from Matt about results of a local build and deploy test.

matthewandrewpalmer commented:
Built successfully and services running correctly when deployed in CDAP.

Returning JSON of company house:
![image](https://user-images.githubusercontent.com/38406765/42226176-3392bcd4-7ed6-11e8-8f8e-6d88fc9bad79.png)
